### PR TITLE
test: Add different types of UIViewController to Swift sample

### LINKED ISSUE #1469: test: Add different types of UIViewControllers to Sample App
Add the following to the iOSSwift sample app with some content:

* NavigationController
* SplitViewController
* TableViewController
* UIViewController initialized with a convenience init

Please add a back button, so the UITests can easier navigate back from each of the UIViewControllers.

If possible also add some of the above view controllers to the tvOS sample.

### PR DESCRIPTION
**Pull request by brustolin**
Added more UIViewController types to our test sample.

Closes #1469, #1470

_#skip-changelog_

### CODE REVIEW

File: Samples/iOS-Swift/iOS-Swift/ViewControllers/TableViewController.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        return 100
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "CELL") ?? UITableViewCell(style: .default, reuseIdentifier: "CELL")
+        
+        let g = Double(indexPath.row) / 99
+        let r = 1.0 - g
+        
+        cell.backgroundColor = UIColor(red: r, green: g, blue: 0, alpha: 1)
```
Comment (philipphofmann) [inline Samples/iOS-Swift/iOS-Swift/ViewControllers/TableViewController.swift:20]: `l`: Can we maybe use a Sentry color here? Red is really aggressive 😁

File: Samples/iOS-Swift/iOS-Swift/ViewControllers/TableViewController.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 100
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "CELL") ?? UITableViewCell(style: .default, reuseIdentifier: "CELL")
+        
+        let g = Double(indexPath.row) / 99
```
Comment (philipphofmann) [inline Samples/iOS-Swift/iOS-Swift/ViewControllers/TableViewController.swift:17]: `m`: I think CI complains about this line of code.

**Review by philipphofmann — APPROVED** [review]
LGTM, but CI is complaining.

File: Samples/iOS-Swift/iOS-Swift/ViewControllers/TableViewController.swift (hunk 1 above)
Comment (brustolin) [reply Samples/iOS-Swift/iOS-Swift/ViewControllers/TableViewController.swift:20]: Maybe a gradient from white to black instead of red to green?

**Pull request merged**